helpers/tempesta.py

```diff
@@ -225,15 +225,19 @@ def get_config(self):
 class TfwLogger(object):
     logger_config: str = tf_cfg.cfg.get("TFW_Logger", "logger_config")
     host: str = tf_cfg.cfg.get("TFW_Logger", "ip")
-    port: int = int(tf_cfg.cfg.get("TFW_Logger", "clickhouse_port"))
+    tcp_port: int = int(tf_cfg.cfg.get("TFW_Logger", "clickhouse_tcp_port"))
```

Comment:
Let's make `tcp_port` variable like property method. I will add `CLICKHOUSE_TCP_PORT` for `tcp_port` on CI (8123 -> 58123). We must not change it in tests.